src/_test_/commentCounter.test.js

```diff
@@ -0,0 +1,26 @@
+import commentsCounter from '../_mock_/commentsCounter.js';
+
+describe('Check Items count number', () => {
+  test('check if the function actually does the count', async () => {
+    const data = await commentsCounter(52795);
+    const count = data.length;
+    expect(count).toEqual(7);
```

Comment:
- [ ] When I run this test, it fails as shown below. Please fix it.

<img width="1159" alt="Screenshot 2021-12-24 at 18 06 42" src="https://user-images.githubusercontent.com/64774936/147366624-05cc7be8-9359-4cf2-9f36-e853dc53639c.png">

Reply:
Hello Ghiftee,
Thanks for your wonderful review. The test fail because you made a comment on the first item which was used for the test case therefore the comment count increase by one making it 8 comments